control-base/src/main/java/fi/nls/oskari/control/layer/SaveLayerHandler.java

```diff
@@ -271,8 +271,8 @@ private boolean handleRequestToMapLayer(final ActionParameters params, OskariLay
             }
         }
 
-        InspireTheme theme = inspireThemeService.find(params.getHttpParam("inspireTheme", -1));
-        ml.addInspireTheme(theme);
+        MaplayerGroup theme = oskariMapLayerGroupService.find(params.getHttpParam("inspireTheme", -1));
```

Comment:
Local name `theme` kind of unfitting, but not too big of a deal. Same thing as with the previously mentioned JSON key thingy, the `inspireTheme` parameter needs some attention once the frontend is updated.